src/styles/components/elements.css

```diff
@@ -3,7 +3,7 @@
     @apply my-4 border-t-divider;
   }
 
-  a:not(.pos-clean) {
+  a:not(.pos-clean):not(.pos-pagination__link) {
```

Comment:
we'll definitely need a solution for this in the long run
for now it's fine but we'll probably want to provide a scope for the default `a` element styles. (like: only if it's in a paragraph / prose, etc)